packages/react-app/src/graphql/subgraph.ts

```diff
@@ -1,5 +1,14 @@
 import { BigNumberish } from "@ethersproject/bignumber";
 
+export interface TournamentCuration {
```

Comment:
this interface is not used anywhere in the frontend, could be removed

Reply:
Yup, I will remove it